### ISSUE TITLE
Required fields

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1908]
> Even though it is common to identify required fields with an asterisk, it should be explicit that asterisk = required. Recommend adding line of text below each form title: "All fields marked with * are required."

## Comments

Agree on this, but this can be done by using the text widget above each form.

**Comment by jlyon** [comment]
@kevindherman this is not going to be easy to do automatically. Recommendation if they want this on Marin DPW: use the description field in the form builder
![Screenshot - 2021-11-17T195257 074](https://user-images.githubusercontent.com/550909/142352437-96514cd7-7960-4fd7-99a7-55a6bc6bcc3b.png)
:
